src/blockchains/ethereum/index.ts

```diff
@@ -17,13 +18,17 @@ let web3;
 let bancorConverter = BancorConverter;
 let contractRegistry = ContractRegistry;
 let registryAbi = BancorConverterRegistry;
+let networkAbi = BancorNetwork;
 let registry;
+let network;
 
 export async function init(ethereumNodeUrl, ethereumContractRegistryAddress = '0xf078b4ec84e5fc57c693d43f1f4a82306c9b88d6') {
     web3 = new Web3(new Web3.providers.HttpProvider(ethereumNodeUrl));
     const contractRegistryContract = new web3.eth.Contract(contractRegistry, ethereumContractRegistryAddress);
     const registryBlockchainId = await contractRegistryContract.methods.addressOf(Web3.utils.asciiToHex('BancorConverterRegistry')).call(); // '0x85e27A5718382F32238497e78b4A40DD778ab847'
+    const networkBlockchainId = await contractRegistryContract.methods.addressOf(Web3.utils.asciiToHex('BancorNetwork')).call(); // '0x85e27A5718382F32238497e78b4A40DD778ab847'
```

Comment:
Is this comment still needed?

Reply:
In the `redesign_and_cleanups` branch - no (and neither is the one above it).
Here - I'm not sure, but it doesn't really matter (will be gone soon either way).

Reply:
ok